### ISSUE TITLE
Amstrad CPC Core (alt), fast FDC lost

### ISSUE THREAD

**Issue by AxelStone** [mist-devel/mist-binaries#131]
Hello, I've seen that in the last version of the Amstrad CPC Core (alt one) the fast FDC controller feature has been lost. No mather which option you select (Original or Fast), the loading times are the same.

Simple test: Gryzor.

- New core with Original FDC: 1:20 min loading time.
- New core with Fast FDC: 1:20 min loading time.
- Old core with Original FDC: 1:20 min loading time.
- Old core with Fast FDC: 0:26 min loading time.

Thanks for your support.

**Comment by gyurco** [comment]
True, with the latest adjustments in accuracy eroded the fast mode. I'll investigate to make it useful again.
